roles/ceph-docker-common/tasks/main.yml

```diff
@@ -21,3 +21,58 @@
 - set_fact:
     monitor_name: "{{ ansible_fqdn }}"
   when: mon_use_fqdn
+
+- name: check if a cluster is already running
+  command: "docker ps -q -a --filter='ancestor={{ ceph_docker_image }}:{{ ceph_docker_image_tag }}'"
+  register: ceph_health
+  changed_when: false
+  failed_when: false
+  always_run: true
+
+- include: checks.yml
+  when:
+    - (not mon_containerized_deployment_with_kv and
+       ((inventory_hostname in groups.get(mon_group_name, [])) or
+        (inventory_hostname in groups.get(nfs_group_name, [])))) or
+      (not osd_containerized_deployment_with_kv and inventory_hostname in groups.get(osd_group_name, []))
+    - ceph_health.rc != 0
+    - not "{{ rolling_update | default(false) }}"
+
+- include: "{{ playbook_dir }}/roles/ceph-common/tasks/misc/ntp_atomic.yml"
+  when:
+    - is_atomic
+    - ansible_os_family == 'RedHat'
+    - ntp_service_enabled
+
+- include: "{{ playbook_dir }}/roles/ceph-common/tasks/misc/ntp_redhat.yml"
+  when:
+    - not is_atomic
+    - ansible_os_family == 'RedHat'
+    - ntp_service_enabled
+
+- include: "{{ playbook_dir }}/roles/ceph-common/tasks/misc/ntp_debian.yml"
+  when:
+    - ansible_os_family == 'Debian'
+    - ntp_service_enabled
+
+- include: "{{ playbook_dir }}/roles/ceph-common/tasks/docker/fetch_image.yml"
```

Comment:
This should definitely be moved into ``ceph-docker-common`` and out of ``ceph-common``. I'm seeing instances now where values defined with ``--extra-vars`` are ignored inside the include from another role. See: https://github.com/ceph/ceph-ansible/pull/1487#issuecomment-298417579

Perhaps this is an ansible bug of some sort related to include usage?